src/Symfony/Component/AssetMapper/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=8.1",
+        "composer/semver": "^3.0",
```

Comment:
New dependency - composer seems like a quite safe one. And I don't want to make this feature opt-in: all users should see warnings about version incompatibilities.

Reply:
It should be added to the main `composer.json` as well, right? Or maybe we don't have tests that covers that?

Reply:
Guilty - a lot moving pieces - thanks for catching these things.